### ISSUE TITLE
dynamic import cannot have trailing comma

### ISSUE THREAD

**Issue by surma** [palantir/tslint#4494]
### Bug Report

- __TSLint version__: 5.12.1
- __TypeScript version__: 3.3.1
- __Running TSLint via__: CLI 

#### TypeScript code being linted

```ts
const x = import(
  "./module",
);
```

with `tslint.json` configuration:

```json
{
    "defaultSeverity": "error",
    "extends": [
        "tslint:recommended"
    ],
    "rules": {
        "trailing-comma": true
    },
}
```

#### Actual behavior

No error.

#### Expected behavior

Trailing commas are invalid in `import()` statements as they are not a function call but a syntactic construct. TypeScript v3.3+ does not compile with a trailing comma in an import statement. See [this PR](https://github.com/Microsoft/TypeScript/pull/29556). This should be an error in tslint, too.

**Comment by adidahiya** [comment]
If it fails to compile in TS 3.3, then adding the same exact check in TSLint feels redundant. Why report it twice? I'm going to mark this as declined for now but open to community feedback.

**Comment by adidahiya** [comment]
Yeah autofixing is a fair argument but IMO the benefits don't outweigh the costs in this one, especially given that it's a formatting rule, a category which we are de-prioritizing in TSLint as a whole (#3592)

**Comment by adidahiya** [comment]
actually, I looked at the rule implementation again and this shouldn't be too hard to add. open to PRs with unit tests

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is being deprecated** and no longer accepting pull requests for major new changes or features. See #4534. 😱

If you'd like to see this change implemented, you have two choices:
* **Recommended:** Check if this is available in [ESLint](https://eslint.org) + [typescript-eslint](https://typescript-eslint.io) ✅
* _Not Recommended: Fork TSLint locally_ 🤷‍♂️

👋 It was a pleasure open sourcing with you!

_If you believe this message was posted here in error, please comment so we can re-open the issue!_

**Comment by JoshuaKGoldberg** [comment]
🤖 Beep boop! 👉  **TSLint is deprecated** 👈 _(#4534)_  and you should **switch to [typescript-eslint](https://typescript-eslint.io)**! 🤖 

🔒 This issue is being locked to prevent further unnecessary discussions. Thank you! 👋
